lib/enum_state_machine/integrations/active_record.rb

```diff
@@ -486,11 +486,11 @@ def initialize(*)
         def define_action_hook
           if action_hook == :save
             define_helper :instance, <<-end_eval, __FILE__, __LINE__ + 1
-              def save(*)
+              def save(**)
```

Comment:
@GabrielaJustine16 What's the minimum Ruby version that supports this syntax?

Reply:
Double-splat goes back to Ruby 2.0.